Musikcube does some stripping which breaks with a custom build directory,
also not necessary as we do stripping ourselves

diff --git CMakeLists.txt CMakeLists.txt.2
index 2dbd1f6e99..35405944b1 100644
--- CMakeLists.txt
+++ CMakeLists.txt.2
@@ -322,8 +322,3 @@ endif()
 add_custom_target(postbuild ALL DEPENDS musikcube musikcubed)
 add_custom_command(TARGET postbuild POST_BUILD COMMAND cmake .)
 
-# strip binaries in release mode
-if (CMAKE_BUILD_TYPE MATCHES Release)
-  message(STATUS "stripping binaries...")
-  add_custom_command(TARGET postbuild POST_BUILD COMMAND ./strip-nix.sh)
-endif()
